src/core/src/op/scatter_update.cpp

```diff
@@ -122,3 +122,59 @@ bool op::v3::ScatterUpdate::has_evaluate() const {
     }
     return false;
 }
+
+namespace {
+bool scatter_label_evaluator(const Node* node, TensorLabelVector& output_labels) {
+    const auto& input_values = node->input_values();
+
+    constexpr auto data_in_idx = 0;
+    constexpr auto updates_in_idx = 2;
+    std::vector<size_t> data_labels = input_values[data_in_idx].get_tensor().get_value_label();
+    std::vector<size_t> updates_labels = input_values[updates_in_idx].get_tensor().get_value_label();
+
+    if (ov::has_no_labels(data_labels) && ov::has_no_labels(updates_labels)) {
+        return false;
+    }
+
+    constexpr auto element_type = (sizeof(size_t) == 8) ? element::u64 : element::u32;
+    std::vector<ov::runtime::Tensor> input_tensors;
+    input_tensors.reserve(input_values.size());
+
+    auto make_input_label = [&](const Output<Node>& input, TensorLabel& labels) {
+        input_tensors.emplace_back(element_type, input.get_shape());
+        labels.resize(shape_size(input.get_shape()));
```

Comment:
Detail or next time
```suggestion
        labels.resize(shape_size(input.get_shape()), ov::no_label);
```
To be sure if new elements have got correct value.